### DISCUSSION TITLE
GitHub Student Application Denied

### DISCUSSION THREAD

**Discussion by carolinaMera15** [community/community#142274] [GitHub Education]
### Hi everyone,

I am applying for the GitHub Student Developer Pack. I am using an official letter from my school, which includes the date and the name of my educational program, but my application has been denied.Anyone else with the same problem?

**Comment by madhawaawishka** [comment]
Hi! I had a similar issue when I first applied. Here are a few things to check and try:

1.Ensure all document details are clear: Make sure the official letter from your school clearly shows your full name, the date, and your enrollment status. Sometimes GitHub needs to verify that the document is official and current, so ensure it's not too old or missing key information.

2.File format: Ensure that the document is uploaded in an acceptable format like PDF or JPG, and that it's clearly legible. GitHub sometimes denies applications due to unclear or unreadable documents.

3.Use a valid school email address: If possible, try using your school-issued email address (with a .edu or similar domain) for the application. It helps confirm your student status more easily.

4.Reapply with a new document: You can try reaching out to your school’s administration to see if they can provide a more detailed or updated letter. If you reapply, GitHub might review it again with fresh eyes.

5.Contact GitHub support: If you're sure all the documents are correct, consider reaching out to GitHub support directly for clarification on why your application was denied. Sometimes there are specific reasons that aren’t immediately clear.

I hope this helps! Good luck!

**Comment by yusufarslandev** [comment]
Hello,
My student ID card has not arrived yet. I printed out my approved document. There is only camera verification on the document approval screen in the application. No matter what I do, I cannot fully scan the document from the webcam. The text is not clear.

Every time I get a similar error, I cannot upload the document.

![image](https://github.com/user-attachments/assets/15352a94-d336-427d-ac57-93c74d5f3bd7)
